README.md

```diff
@@ -1,24 +1,163 @@
 
-* Services (job queues, cache servers, search engines, etc.)
+**[My Blog]** is a Ruby on Rails application that creates a blog. This blog is a fully functional website that will show the list of posts and empower readers to interact with them by adding comments and liking posts.
 
-* Deployment instructions
+## 🛠 Built With <a name="built-with"></a>
 
-* ...
+### Tech Stack <a name="tech-stack"></a>
+
+- Languages: Ruby, SQL
+- Frameworks: Rspec
+- Technologies Used: GIT, GITHUB, LINTERS
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Used database migration files to maintain database schema
+- Used Ruby Gems as a software packages system.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ ruby installed
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git https://github.com/jonas-45/my-blog-rails.git
+  cd my-blog-rails
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd my-blog-rails
+  bundle install
+```
+
+### Usage
+
+To run the project, start rails server:
+
+```sh
+  rails server
+  Go to localhost:3000 on your broswer
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rspec spec spec\
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Wullah Jonas Nuta**
+
+- GitHub: [@jonas-45](https://github.com/jonas-45)
+- Twitter: [@jonaswullah](https://twitter.com/jonaswullah)
+- LinkedIn: [jonas-wullah](https://linkedin.com/in/jonas-wullah)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Create a data model.]**
+- [ ] **[Add validations and Model specs]**
+- [ ] **[Add forms]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/ambrose-kibet/Catalog-of-my-things/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it a thumps up and follow us for more updates
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Thanks to @jonas-45, @Yemisirach and @ambrose-kibet for all the support and hardwork put into this project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] It would be better if we fix the MIT license broken link, currently the link returns `404` error page not found as we can see in the screenshot below
![Screenshot from 2023-05-17 10-21-07](https://github.com/jonas-45/my-blog-rails/assets/46388113/4ca947ff-a117-4a41-a3dc-b5a49f525b40)

Reply:
Thanks @addod19  for going through my work. I will work on it right away.